TAGradingServer/account/admin-latedays-exceptions.php

```diff
@@ -0,0 +1,752 @@
+<?php
+
+//Author: Peter Bailie, Systems Programmer, RPI Computer Science, July 2016
+
+/* MAIN ===================================================================== */
+
+include "../header.php";
+
+check_administrator();
```

Comment:
Yeah, it's something that should be removed from all pages most likely, but not worth the effort at this point. You don't really need to change anything here, I'm just stating that the else branch is impossible to execute (so there's no way to test it).